src/tests/ftest/launch.py

```diff
@@ -2202,6 +2194,15 @@ def process(self, test, repeat, stop_daos, archive, rename, jenkinslog, core_fil
                 "depth": 1,
                 "timeout": 900,
             }
+
+            # Get the core file pattern information
+            try:
+                core_files = self._get_core_file_pattern(
+                    test.host_info.servers.hosts, test.host_info.servers.hosts, process_cores)
```

Comment:
We should be checking all hosts for core files not just the hosts currently selected to act in the server role:
```suggestion
                    test.host_info.servers.hosts, test.host_info.clients.hosts, process_cores)
```

Reply:
Done